tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -829,7 +829,7 @@
   },
   {
     "id": "{{.ok}} deleted action {{.name}}\n",
-    "translation": "{{.ok}} deleted action {{.name}}\n"
+    "translation": "{{.ok}} deleted action '{{.name}}'\n"
```

Comment:
I don't think we need to quote entity names that are displayed in bold font. Here is what it looks like:
ok: created action **'actionName'**

@mdeuser, what do you think?